loadtest/control/simulcontroller/controller.go

```diff
@@ -302,3 +371,42 @@ func (c *SimulController) sendFailStatus(reason string) {
 func (c *SimulController) sendStopStatus() {
 	c.status <- control.UserStatus{ControllerId: c.id, User: c.user, Info: "user stopped", Code: control.USER_STATUS_STOPPED}
 }
+
+// InjectAction allows a named UserAction to be injected that is run once, at the next
+// available opportunity. These actions can be injected via the coordinator via
+// CLI or Rest API.
+func (c *SimulController) InjectAction(actionID string) control.UserActionResponse {
```

Comment:
I may have missed this but is there any special reason for not returning a simple `error` from this method? Given the action is run asynchronously I am not sure the added complexity of the wrapper is necessary.

Reply:
I think I was following a pattern from elsewhere in the app.  Fixed.